src/sidebar/components/thread-list.js

```diff
@@ -159,7 +174,9 @@ function ThreadList({ thread }) {
     setThreadHeights(prevHeights => {
       const changedHeights = {};
       for (let { id } of visibleThreads) {
-        const threadElement = document.getElementById(id);
+        const threadElement = /** @type {HTMLElement} */ (document.getElementById(
```

Comment:
I can't see any of these elements as being null, but I suppose it's possible? However, I felt a cast was a reasonable risk here.

Reply:
Yes, this seems fine.